projects/teleport/project.yaml

```diff
@@ -1,10 +1,11 @@
 homepage: "https://github.com/gravitational/teleport"
-main_repo: "https://github.com/gravitational/teleport"
+language: go
 primary_contact: "reed@goteleport.com"
-auto_ccs :
-  - "adam@adalogics.com"
```

Comment:
Any specific reason to remove this contact from auto_cc?
cc @AdamKorcz

Reply:
We overhauled our entire fuzzing setup and started all three required integration files from scratch. Since @AdamKorcz isn't involved in @gravitational and the setup he added in https://github.com/google/oss-fuzz/pull/4570 is obsolete now, just didn't include him. However, I don't have any objections to keeping him listed, so if that speeds up the landing process, happy to re-add him. Just wanting to get this PR landed soon so we can begin fuzzing.

Reply:
Thanks for the heads up, @Navidem. I helped with the initial integration (https://goteleport.com/blog/fuzzing), and I am happy if the  Teleport are taking over the work.

Reply:
Thanks @reedloden and @AdamKorcz. I will merge this now.